contribution_evaluation.py

```
@@ -0,0 +1,304 @@
+
+
+# get origin files from pr
+def get_original_files_from_pr(owner, repo, pr_number):
+    # get data from PR
+    pr_info = get_pr_info(owner, repo, pr_number).json()
+    if not pr_info:
+        return []
+
+    # get the sha of head from PR
+    head_sha = pr_info["head"]["sha"]
+    head_ref = pr_info["head"]["ref"]
```

Ruff (0.8.2)

Local variable `head_ref` is assigned to but never used

Remove assignment to unused variable `head_ref`

(F841)